brainiak/fcma/voxelselector.py

```diff
@@ -91,7 +91,11 @@ class VoxelSelector:
     process_num: Optional[int]
         The number of processes used in cross validation.
         If None, the number of processes will equal
-        the number of hardware threads.
+        the number of hardware threads,
+        i.e. the number returned by cpu_count().
+        If 0, cross validation will not use python multiprocessing.
+        NOTE: on some compute cluster, python multiprocessing launched
+        via MPI on a remote machine may result in child processes disfunct.
```

Comment:
I am not sure what you mean by "disfunct". Can you please rephrase this? Most importantly, please link to a description of the issue, perhaps even create a new issue if it is not yet clear what is happening.

Finally, please make this note using the `Note` RST syntax, so that it stands out in the HTML docs, like in SRM:
http://pythonhosted.org/brainiak/brainiak.funcalign.html#brainiak.funcalign.srm.SRM.random_state_
https://github.com/IntelPNI/brainiak/blob/master/brainiak/funcalign/srm.py#L81

Reply:
Defunct... Sorry for the typo

I cannot find a reference to this issue, but will follow up with the TACC guys. The `Note` syntax is fixed.